x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_details/format_values.js

```diff
@@ -63,7 +67,16 @@ export function formatValues([key, value]) {
     // numbers rounded to 3 decimal places
     case 'average_search_time_per_bucket_ms':
     case 'exponential_average_search_time_per_hour_ms':
-      value = typeof value === 'number' ? roundToDecimalPlace(value, 3).toLocaleString() : value;
+      value = roundTo3DecimalPlace(value);
+      break;
+    case 'total_bucket_processing_time_ms':
+      value = roundTo3DecimalPlace(value);
+      break;
+    case 'average_bucket_processing_time_ms':
+      value = roundTo3DecimalPlace(value);
+      break;
+    case 'exponential_average_bucket_processing_time_ms':
```

Comment:
Looks like we should round to 3 decimal places for `exponential_average_bucket_processing_time_per_hour_ms` too:

![image](https://user-images.githubusercontent.com/7405507/80736202-b4a68580-8b09-11ea-871f-ad6c2947e8f5.png)